bin/docpress

```diff
@@ -29,6 +35,23 @@ function run () {
   }
 }
 
+function help () {
```

Comment:
A more clean would be:

- put the content of the help command in a file, for example: https://github.com/bumped/bumped/blob/master/bin/help.txt
- load the file!
https://github.com/bumped/bumped/blob/master/bin/index.js#L10